pkg/web/context.go

```diff
@@ -74,6 +73,11 @@ func (ctx *Context) run() {
 	}
 }
 
+func (ctx *Context) Invoke(h http.Handler) ([]reflect.Value, error) {
```

Comment:
Since `inject.go` is removed, do we still need these returns on the function signature? In other words, does it fulfill a interface we still need?

Reply:
nice catch! in fact, we do not need this function at all. removed in 2568aa5